STYLE.md

```diff
@@ -0,0 +1,12 @@
+# Go coding style
+
+* The xDS Relay source code is formatted using
+  [golangci-lint](https://github.com/golangci/golangci-lint). Thus all white spaces, etc.
```

Comment:
Can we add all the linters we will be using here also or link to the golangci configuration file?